Automattic/wp-calypso
# Contact Form: Add a notice of reduced chat availability

#### Changes proposed in this Pull Request

Adds a notice in the Contact Forms that we expect chat availability to be reduced for a few days. Expiration is set to a year from now, since we'll re-evaluate next week if it should come down.

<img width="749" alt="Screen Shot 2020-03-27 at 3 55 18 PM" src="https://user-images.githubusercontent.com/518059/77800224-4ff89680-7044-11ea-8ad1-b71a595184b9.png">
<img width="336" alt="Screen Shot 2020-03-27 at 3 55 29 PM" src="https://user-images.githubusercontent.com/518059/77800228-51c25a00-7044-11ea-9ef2-bc7cf623fe7c.png">


#### Testing instructions

Verify that nothing breaks with the current notice. Also, test the vanilla `ContactFormNotice` and the updated `LiveChatClosureNotice`s to make sure they still work as expected.
